src/main/webapp/js/user-page-loader.js

```diff
@@ -33,7 +33,7 @@ function setPageTitle() {
  * Shows the message form if the user is logged in and viewing their own page.
  */
 function showMessageFormIfViewingSelf() {
-  fetch('/login-status')
+  fetch('/MessageServlet')
```

Comment:
I think this is not right because to verify if the user has has signed in, we want to fetch login-status and not the MessageServlet (even if we want to fetch MessageServlet it should be with its annotation like: '/messages')

Reply:
Let me know if I am missing something here, and you think that we need to fetch MessageServlet :)